### DISCUSSION TITLE
high availability ?

### DISCUSSION THREAD

**Discussion by Sleepful** [pockethost/pockethost#406] [Q&A]
the main issue with PB is that it runs on a single machine, so if the network or the machine goes down, the whole thing dies.

Does pockethost has any affordances to remediate this issue? 

Cheers

**Comment by benallfree** [comment]
PocketHost does not implement a failover strategy, though that could be an interesting project.

**Reply by Sleepful** [reply]
It would be a very interesting project indeed.

Any kind of SLA-type guarantees? 

Or statistics of past performance?

**Reply by benallfree** [reply]
We don't offer an SLA because it's just me running everything :)

Uptime is around 99.95% annual average, and the last few months have been 100%.
